Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM hydroshare/hs_docker_base:hydro-flower
+FROM hydroshare/hs_docker_base:release-1.10.0
```

Comment:
No idea why this is showing up, the base version has already been updated in develop.  I must've made a commit in this branch before develop was updated.